ingest/src/test/scala/hydra/ingest/http/IngestionEndpointSpec.scala

```diff
@@ -236,6 +238,17 @@ class IngestionEndpointSpec
         responseAs[String] should include(s"Schema '$topic' cannot be loaded. Cause: hydra.avro.resource.SchemaResourceLoader$$SchemaNotFoundException: Schema not found for $topic")
       }
     }
+  }
 
+  "The V2 Ingestion path" should {
+    "Return 200" in {
+      val validation = RawHeader(HYDRA_VALIDATION_STRATEGY, "relaxed")
+
+      val request = Put("/v2/topics/exp.blah.blah/records", """{"test":true, "extraField":true}""").withHeaders(validation)
```

Comment:
Nit: I'd say we should just leave the validation out of this test case so that it is as minimal as possible to test what we are actually looking for. If we do need a test for the validation strategy here, then we should create another test case

Reply:
Fair enough. I believe it was a copy paste, and did not consider that.

Reply:
I figured, it isn't a big deal, I am just being pedantic here. Feel free to leave it or not, but just a general principle to consider in test cases